src/registrar/forms/domain.py

```diff
@@ -230,6 +232,29 @@ def __init__(self, *args, **kwargs):
         self.fields["email"].error_messages = {
             "required": "Enter an email address in the required format, like name@example.com."
         }
+        self.domainInfo = None
+
+    def setDomainInfo(self, domainInfo):
+        """Set the domain information for the form.
+        The form instance is associated with the contact itself. In order to access the associated
+        domain information object, this needs to be set in the form by the view."""
+        self.domainInfo = domainInfo
```

Comment:
(Super minor nitpick / optional) This is using camelCase rather than snake_case

Reply:
fixed, nice catch